lib/session_setup/view/session_setup_page.dart

```diff
@@ -104,20 +114,59 @@ class PromptCard extends StatelessWidget {
   final bool selected;
   final VoidCallback onTap;
 
+  @override
+  State<PromptCard> createState() => _PromptCardState();
+}
+
+class _PromptCardState extends State<PromptCard> {
+  bool _hover = false;
+
+  static const _normalSize = Size(240, 120);
+  static const _effectSize = Size(260, 140);
+
   @override
   Widget build(BuildContext context) {
+    final isBig = _hover || widget.selected;
     return InkWell(
-      onTap: onTap,
-      child: Container(
-        width: 240,
-        height: 120,
-        padding: const EdgeInsets.all(8),
-        child: Card(
-          color: selected ? Theme.of(context).primaryColor : null,
-          child: Center(
-            child: Text(
-              prompt,
-              textAlign: TextAlign.center,
+      onTap: widget.onTap,
+      child: MouseRegion(
+        onEnter: (_) {
+          setState(() {
+            _hover = true;
+          });
+        },
+        onExit: (_) {
+          setState(() {
+            _hover = false;
+          });
+        },
+        child: SizedBox(
+          width: _normalSize.width,
+          height: _normalSize.height,
+          child: OverflowBox(
+            maxWidth: _effectSize.width,
+            maxHeight: _effectSize.height,
+            child: AnimatedSize(
```

Comment:
This animation definitely didn't worked as I expected, for some reason once I have an `OverflowBox`, the animation stops to work, maybe someone has any idea on why?

Reply:
You could try to just wrap your card or whatever child widget with an animated container with a transformed scale on it instead. 👀

Reply:
Ohhh, this is way better! Updated